examples/string-templates/string_templates.bal

```diff
@@ -0,0 +1,26 @@
+import ballerina/io;
+
+public function main() {
+    // Note how the first `\n` is treated as is.
+    string s1 = string `line one \n rest of line 1 ${"\n"} second line`;
+    io:println(s1);
+
+    // Use interpolations to add the ` and  $ characters.
+    string s2 = string `Backtick: ${"`"} Dollar: ${"$"}`;
+    io:println(s2);
+
+    // You can use an empty string interpolation to break a template expression
```

Comment:
Decided to keep this as is since I believe it was an answer to a question in discord asking specifically for breaking template expression across multiple lines

Reply:
I would switch the two examples.

A string template expression can be written on multiple lines by breaking at interpolations.

And then say

If there are no interpolations to break at a required point, you can introduce an interpolation with  and empty string.